Examples/FluentSpreadsheets.Examples.Components/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Drawing;
+using FluentSpreadsheets;
+using FluentSpreadsheets.Styles;
+using static FluentSpreadsheets.ComponentFactory;
+
+var helloStacks = VStack
+(
+    HStack
+    (
+        Label("Hello"),
+        Label(",")
+    ),
+    Label("Stacks!")
+);
+
+var helloStyles = VStack
+(
+    HStack
+    (
+        Label("Hello").WithContentAlignment(HorizontalAlignment.Trailing),
+        Label(",")
+    ),
+    Label("Styles!").WithContentAlignment(HorizontalAlignment.Center, VerticalAlignment.Top)
+).WithTrailingBorder(BorderType.Thin, Color.Black).WithBottomBorder(BorderType.Thin, Color.Black);
```

Comment:
This example contains the code from documentation, the section describing component creation. It would be easier to have it here, and change it, when api changes, and copy it into the docs, rather than reproduce it every single time in playground, before copying it (it won't compile if it is not conforming to an api). 
I think it is highly unlikely for someone to assume that you need to build a component composition from scratch it order to apply styles, or get modified version without modifying base object. Fluent API's is basically a declarative API, and declarative syntax implies immutability. 
But it can be unobvious for someone not familiar with concepts, so components' immutable nature should be noted in docs.